### DISCUSSION TITLE
Pagination has stopped working

### DISCUSSION THREAD

**Discussion by GlenWarren** [orchidsoftware/platform#2455] [Q&A]
I'm not sure why, but pagination has stopped working for us.

When clicking on (or hovering over) the next page number, the url that we are directed to is `http://localhost/?page=2` instead of `http://localhost:5000/admin/users?page=2`. This is happening in all our environments, not just locally.

I can't figure out what's changed for this to start happening now. It's as if Orchid is no longer compatible with eloquent's `paginate` methods.

In the IndexScreen file, I am applying pagination like this:

```
    public function query() : array
    {
        return [
            'collection' => User::filters()->paginate(10)
        ];
    }
```

I have the same problem when I use `simplePaginate()`.

When I navigate manually to the correct url (`http://localhost:5000/admin/users?page=2`) the results on my screen remain the same, as if I am still on page 1. It does not seem to even recognise/process the `page` param.

Anyone have any ideas?

**Comment by tabuna** [comment]
If you think the error is on the package side. Then please create an issue with steps to reproduce the problem. 

Because this is not typical behavior.

**Reply by GlenWarren** [reply]
We have fixed the issue.
It seems that the issue was caused by the `PaginatorServiceProvider` not registering. We have manually registered the provider, and this has fixed the issue.
We think this is due to us using Laravel RoadRunner.
